### PR TITLE
add release 4.07.1

### PR DESCRIPTION
**Pull request by damiendoligez**
As for #1009, I don't really know what I'm doing, so please review and amend as needed.

### CODE REVIEW

File: site/releases/4.07.1.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+<!-- ((! set title OCaml 4.07.1 !)) -->
+
+# OCaml 4.07.1
+This page describes OCaml version **4.07.1**, released on
+2018-07-10.  Go [here](./) for a list of all releases and
```
Comment (pmetzger) [inline site/releases/4.07.1.md:5]: Was it really released July 10? I think that's 4.07.0, yes?

File: site/releases/4.07.1.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+Alternative Compilers
+---------------------
+
+Additionally, the following projects allow you to compile OCaml code to
+targets traditionally associated with other languages:
+
+* [Js_of_ocaml](http://ocsigen.org/js_of_ocaml/) is a stable OCaml
+  to JavaScript compiler.
+* [Bucklescript](http://bucklescript.github.io/bucklescript/) is a
+  newer OCaml to JavaScript compiler. See its
```
Comment (pmetzger) [inline site/releases/4.07.1.md:64]: I note that Bucklescript does not support 4.07.1

File: site/releases/4.07.1.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  [tarball](http://caml.inria.fr/pub/distrib/ocaml-4.07/ocaml-4.07-refman.info.tar.gz)
+  of Emacs info files,
+
+
+![](../img/source.gif "") Download directory
+--------------------------------------------
+
+All the files are also available in [this
+directory](http://caml.inria.fr/pub/distrib/ocaml-4.07), along with
+their MD5 and SHA512 checksums.
```
Comment (pmetzger) [inline site/releases/4.07.1.md:98]: Probably MD5 checksums are obsolete at this point. SHA-2 and SHA-3 checksums seem reasonable.

**Comment by Chris00** [comment]
Thanks for your work — your submission is fine, I just did minor tweaks following @pmetzger remarks.

**Pull request merged**